### ISSUE TITLE
 SCE-internals man page 

### ISSUE THREAD

**Issue by FelixErnst** [drisso/SingleCellExperiment#54]
Hi,

I stumbled on the `SCE-internals` man page to look for infos on the `int_*` slots.

For both getters and setters is states something like this:

```
int_elementMetadata(x):
Returns a DataFrame of internal row metadata, with number of rows equal to nrow(x). This is analogous to the user-visible colData.
```

Shouldn't that read "This is analogous to the user-visible **rowData**"?

Thanks for any advice on the internal structure of `SingleCellExperiment`

Felix

**Comment by LTLA** [comment]
Yes, that's probably true.

**Comment by FelixErnst** [comment]
also in the Setters section

```
int_elementMetadata(x) <- value:
Replaces the internal row metadata with value, a DataFrame with number of rows equal to nrow(x). This is analogous to the user-visible colData<-.
```

**Comment by FelixErnst** [comment]
@LTLA fyi see above

**Comment by LTLA** [comment]
Yes, fixed yesterday in d669ff74918f482dabea97cedcc85e5bd15f068c.
